### PR TITLE
#723: [Bug] Unable to get Group epoch

### PR DESCRIPTION
**Pull request by pulkitkothari**
#723: [Bug] Unable to get Group epoch

### CODE REVIEW

**Review by raphaelrobert — CHANGES_REQUESTED** [review]
Thanks for the PR! Could make two separate PRs, one for each issue? That way we can discuss this separately.

**Comment by pulkitkothari** [comment]
@raphaelrobert  Done this diff just have one bug fix.

File: openmls/src/group/mls_group/mod.rs [hunk 1]
```diff
@@ -234,6 +234,11 @@ impl MlsGroup {
         self.group.group_id()
     }
 
+    /// Return the epoch
+    pub fn epoch(&self) -> GroupEpoch {
+        return self.group.context().epoch();
```
Comment (raphaelrobert) [inline openmls/src/group/mls_group/mod.rs:239]:
```suggestion
        self.group.context().epoch()
```

**Review by raphaelrobert — CHANGES_REQUESTED** [review]
Thanks, just one nit to make it more rusty.

**Comment by pulkitkothari** [comment]
Still getting used to rust ;p thanks for the review and prompt replies.

**Review by raphaelrobert — APPROVED** [review]
Thanks!

**Pull request merged**